jfxtras-controls/src/main/java/jfxtras/internal/scene/control/skin/CalendarTextFieldSkin.java

```diff
@@ -67,6 +67,13 @@
  */
 public class CalendarTextFieldSkin extends SkinBase<CalendarTextField>
 {
+    
+    private TextField textField = null;
+    private ImageView imageView = null;
+    private GridPane gridPane = null;
+    final public BooleanProperty focusForwardingProperty = new SimpleBooleanProperty();
+    private final Popup popup = new Popup();
```

Comment:
You always create a popup, not every TextField on the screen may actually popup, so with memory considerations on smaller IoT hardwar in mind, I prefer to use a if (popup==null) popup = new Popup() line.